lib/mongoose_push_web/schemas/request/send_notification/deep/alert_and_data.ex

```diff
@@ -0,0 +1,12 @@
+defmodule MongoosePushWeb.Schemas.Request.SendNotification.Deep.AlertAndData do
+  require OpenApiSpex
+
+  OpenApiSpex.schema(%{
+    title: "Request.SendNotification.Deep.AlertAndData",
+    description: "In this request one can pass both alert and data fields.",
+    allOf: [
+      MongoosePushWeb.Schemas.Request.SendNotification.Deep.Alert,
```

Comment:
This won't work. We cannot refer both `Deep.Alert` and `Deep.Data`, assuming those are full schemas that include `Deep.Base`. If request would pass both `Deep.Alert` and `Deep.Data` validation, this would mean, that both of those schemas allow additional properties, which we cannot allow, as for validation, we *need* this process to be not-ambiguous (additionalProperties: true would allow some requests to be validated against more then one of those schemas, and thus the output struct generated by validator would be essentially random.

Reply:
We discussed 'offline' we rollback to previous impl. due to framework constraints we hit here.